practices/pull_requests.md

```diff
@@ -78,10 +78,15 @@ Pull request can also trigger CI/CD to make sure all automated tests pass before
 
 - Make sure that the pull requests are done regularly in your team and that nobody waits way too long for their code to be reviewed. If the code is left unchecked, it can lead to “long feedback loops” and it can effect team motivation.
 - Always have at least two possible approvers for each pull request. Having only one person that can merge pull requests could lead to power hoarding.
-- Encourage small pull requests. Nobody wants to review 5000+ lines of changes within one PR.
+- Encourage short pull requests, preferably not more than 250 lines changed. Large code changes make finding bugs much more difficult and hinder the code review process. A way to minimise large pull requests is to practice breaking down large features into smaller and independent tasks that can be done individually. This also allows different developers to work on issues simultaneously, speeding up product development.
```

Comment:
A thing for consideration: Do you think that using PR templates as a practice is worth mentioning? I personally think so, since it simplifies creating good descriptions. What do you think about it? 

https://medium.com/@team_62166/how-we-learned-to-stop-worrying-and-love-pull-request-templates-and-why-you-should-too-7ed883d8f54e

Reply:
Thanks @ridlees for the review :smile:. 

I do think that a PR template can help make pull requests better given that it is a layout of what a pull request should describe in the least, however I don't think it helps a pull request if maybe the pull request doesn't have a good title or if the description itself is vague. The author of the pull request is really the only person who can ensure that the PR has all the info that a reviewer might need, a PR template can only help so much.

Reply:
Thanks for your point of view. Based on this exchange, I would also add that using PR templates can help (since they work as a structural help, they provide clear instruction and can help with PR management (I quickly see what the PR is about (if the author follows my template or I quickly see that they did not follow and that I need to spent more time on it).

Reply:
@ridlees so should I add another point explaining the use of pull request templates? I definitely see now how it could be useful.

Reply:
I think it can be useful as well. If you don't mind, I would kindly ask you to add it :)

Reply:
Will be making the update soon